commit 31f08672b61d3524126d9842ed307b3d05ae4717
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Thu Mar 23 10:06:56 2017 +0000

    ASoC: Add prompt for ICS43432 codec
    
    Without a prompt string, a config setting can't be included in a
    defconfig. Give CONFIG_SND_SOC_ICS43432 a prompt so that Pi soundcards
    can use the driver.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/sound/soc/codecs/Kconfig b/sound/soc/codecs/Kconfig
index 2b331f7266ab..4ba304870ce6 100644
--- a/sound/soc/codecs/Kconfig
+++ b/sound/soc/codecs/Kconfig
@@ -583,7 +583,7 @@ config SND_SOC_HDAC_HDMI
 	select HDMI
 
 config SND_SOC_ICS43432
-	tristate
+	tristate "InvenSense ICS43432 I2S microphone codec"
 
 config SND_SOC_INNO_RK3036
 	tristate "Inno codec driver for RK3036 SoC"
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1031_31f08672b61d3524126d9842ed307b3d05ae4717.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1031_31f08672b61d3524126d9842ed307b3d05ae4717.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
